build.sbt

```diff
@@ -3,10 +3,26 @@ lazy val scala211 = "2.11.12"
 
 ThisBuild / scalaVersion := scala212
 
+lazy val commonSettings = List(
+  scalacOptions ++= Seq(
+    "-encoding", "utf8",
+    "-deprecation",
+    "-unchecked",
+    "-Xlint",
+  ),
+  scalacOptions ++= (scalaVersion.value match {
```

Comment:
Why this condition? The flag also exists in 2.11 and we don't build for 2.10, right?

Reply:
I took it from here: http://eed3si9n.com/stricter-scala-with-xlint-xfatal-warnings-and-scalafix I guess the reasoning was not to fail on things that are warnings in scala 2.11, but fine in 2.12 (if that exists). I'll remove it; we can always re-introduce it if it becomes an issue.